build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
   id 'org.owasp.dependencycheck' version '5.3.2.1'
   id 'org.sonarqube' version '2.8'
   id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-  id 'org.springframework.boot' version '2.2.7.RELEASE'
+  id 'org.springframework.boot' version '2.3.0.RELEASE'
```

Comment:
This breaks the health checks library at the moment + fixes and adding a secret are really operations that belong to separate PRs

Reply:
Hi @luigibk Let me separate out the PR's